.github/workflows/check-catalyst.yaml

```diff
@@ -511,6 +511,9 @@ jobs:
         echo "MLIR_LIB_DIR=$(pwd)/llvm-build/lib" >> $GITHUB_ENV
 
     - name: Run Python Pytest Tests (backend=lightning.kokkos)
+      env:
+        OMP_PROC_BIND: false
+        OMP_NUM_THREADS: 2
```

Comment:
Could this be handled from within the tests themselves? The problem is that there are other actions that would need updating as well

Reply:
Let's see.

Reply:
Works.